color_helper.py

```diff
@@ -1001,15 +1001,30 @@ def is_enabled(self, mode, palette_name=None, color=None, auto=False):
         )
 
 
-class ColorHelperFileIndexCommand(sublime_plugin.TextCommand):
-    """Color Helper file index command."""
+class ColorHelperFileIndexCommand(sublime_plugin.WindowCommand):
+    """Forces Color Helper to index the colors in the current file.
 
-    def run(self, edit):
+    If `scan_scopes` argument is provided, instead of using the normal
+    configuration it will look for colors inside the given scopes.
+    """
+
+    def run(self, scan_scopes=None, allowed_colors=["all"], use_hex_argb=False):
```

Comment:
Let's not use mutable defaults. Set to none, and if none you can change it to `["all"]`.

Reply:
Or use a tuple as default as those are immutable (and the value isn't changed during runtime).

Reply:
Yup, that works too.

Reply:
I choose the tuple option because I find it more explicit